packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -688,7 +688,12 @@ abstract class Element implements BuildContext {
   Widget get widget => _widget;
   Widget _widget;
 
-  bool _active = false;
+  RenderObjectOwner _owner;
+  RenderObjectOwner get owner => _owner;
+  void set owner(RenderObjectOwner value) {
```

Comment:
i think this should probably be a private API, which only the `RenderObjectToWidgetElement` sets (and which it sets to its render object's owner)